### ISSUE TITLE
Add the gangsta upload

### ISSUE THREAD

**Issue by NoobTracker** [JeffaloBob/scratch-image-uploader#23]
Jeffalo and I found out that there's a way of using assets.scratch.mit.edu in forums. (btw, @JeffaloBob , has lefty discovered that before us? His uploader has that) So there's the suggestion to add SVG support - that's something thumbnails can't do but assets can.
So why don't we just replace the old upload? Well, while the thumbnail thing is not really fixable (and also not exactly not wanted because it's an easy way of using your project thumbnails in your signature), the assets exploit is more like the wikimedia redirect exploit - it may be fixed, so having both is'nt the worst thing in the world. It would be like the normal upload, just with that 8-bit sunglasses over the button (because gangsta).
Alternative: When an SVG is selected, it uses the gangsta methode, otherwise it does everything as normal so it doesn't require an extra button, or more precisely: When you want to upload something that's not .jpg, .png or one of the other supported formats, then it uses gangsta upload because [you may want to upload something else](https://assets.scratch.mit.edu/6192d9b260e30cac4f66006f31958c47.wav). But an extra button would probably be the better solution.

**Comment by JeffaloBob** [comment]
yeah "gangsta" upload is avaliable in the scratchaddons version as of now :)

**Comment by NoobTracker** [comment]
Yay!

**Comment by NoobTracker** [comment]
Has it that stupid sunglasses?
